crypto/fipsmodule/kem/internal.h

```diff
@@ -53,7 +62,9 @@ typedef struct {
 struct kem_key_st {
   const KEM *kem;
```

Comment:
`KEM` object encodes the length of parameters for a particular KEM parameter set. Maybe those could be used?